### PR TITLE
fixed imports for adafruit_minimqtt

### PR DESCRIPTION
**Pull request by codingbandit**
Fixing imports for adafruit_minimqtt re: issue #22

### CODE REVIEW

File: adafruit_azureiot/base64.py [hunk 1]
```diff
@@ -46,8 +46,8 @@
 def _bytes_from_decode_data(data: str):
     try:
         return data.encode("ascii")
-    except:
-        raise ValueError("string argument should contain only ASCII characters")
+    except Exception as exc:
```
Comment (brentru) [inline adafruit_azureiot/base64.py:49]: `as e` please

File: adafruit_azureiot/quote.py [hunk 2]
```diff
@@ -128,7 +128,7 @@ class Quoter(defaultdict):
     # of cached keys don't call Python code at all).
     def __init__(self, safe):
         """safe: bytes object."""
-        super(Quoter, self).__init__()
```
Comment (brentru) [inline adafruit_azureiot/quote.py:131]: Why are you removing `Quoter` from the superclass?

File: adafruit_azureiot/device_registration.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ class DeviceRegistrationError(Exception):
     """
 
     def __init__(self, message):
-        super(DeviceRegistrationError, self).__init__(message)
```
Comment (brentru) [inline adafruit_azureiot/device_registration.py:52]: Why are you removing `DeviceRegistrationError` from the superclass?

**Review by brentru — CHANGES_REQUESTED** [review]
@codingbandit  Thanks for tackling this - I have a few questions...

File: adafruit_azureiot/base64.py (hunk 1 above)
Comment (codingbandit) [reply adafruit_azureiot/base64.py:49]: I originally had e but the lingering the GitHub action wanted more characters :/

File: adafruit_azureiot/quote.py (hunk 2 above)
Comment (codingbandit) [reply adafruit_azureiot/quote.py:131]: Again, this was to appease the linter, it wanted the super class calls to use python 3 syntax

File: adafruit_azureiot/device_registration.py (hunk 3 above)
Comment (codingbandit) [reply adafruit_azureiot/device_registration.py:52]: Same as above

**Comment by codingbandit** [comment]
@brentru - the majority of the changes (other than the import fix) were to pass the linting that occurs in the GitHub action. If desired I can remove those changes, but then the checks will fail on the PR. Thanks :)

File: adafruit_azureiot/base64.py (hunk 1 above)
Comment (brentru) [reply adafruit_azureiot/base64.py:49]: Ok, pylint does specify that.

File: adafruit_azureiot/quote.py (hunk 2 above)
Comment (brentru) [reply adafruit_azureiot/quote.py:131]: kk - np, this shouldn't effect functionality, I've converted super() calls to Python3 syntax before.

**Review by brentru — APPROVED** [review]

**Pull request merged**